kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperation.scala

```diff
@@ -158,6 +151,13 @@ abstract class KyuubiOperation(session: Session) extends AbstractOperation(sessi
         }
       }
     }
+    try {
+      // For launch engine operation, we use OperationLog to pass engine submit log but
+      // at that time we do not have remoteOpHandle
+      getOperationLog.foreach(_.close())
```

Comment:
what happens if this operation is closed ?

Reply:
Sorry, I replied to your message but forgot to submit it